vertical-pod-autoscaler/pkg/updater/logic/updater.go

```diff
@@ -85,6 +108,18 @@ func (u *updater) RunOnce(ctx context.Context) {
 	timer := metrics_updater.NewExecutionTimer()
 	defer timer.ObserveTotal()
 
+	if u.useAdmissionControllerStatus {
+		s, err := u.statusClient.GetStatus()
+		if err != nil {
+			klog.Errorf("Admission Controller status getting error: %v. Skipping eviction loop", err)
+			return
+		}
+		if !status.IsStatusValid(s, time.Minute) {
+			klog.Warningf("Admission Controller status %v is not valid. Skipping eviction loop", s)
```

Comment:
Do you think printing only the Lease will be enough for debugging? Maybe we can also return from isValidStatus an easily parsable reason why it is invalid?

Reply:
Currently we don't have access to the lease. I'm printing timeout duration.